CMakeLists.txt

```diff
@@ -7,6 +7,12 @@
 cmake_minimum_required(VERSION 3.3)
 project(DCA++ C CXX)
 
+################################################################################
+# Set any policies that might be needed
+if(POLICY CMP0074)
+  cmake_policy(SET CMP0074 NEW)
+endif()
+
```

Comment:
I am not very fluent in CMake. What are these 3 lines supposed to do?

Reply:
It tells cmake to pay attention and not ignore -D<PACKAGENAME>_ROOT or PACKAGENAME_ROOT env variables.  I've noticed it is not always obeyed by the find packages, but apparently it works for hdf5.